apps/rush-lib/src/logic/base/BaseLinkManager.ts

```diff
@@ -157,12 +168,20 @@ export abstract class BaseLinkManager {
       }
     }
 
+    // ensure that the last-install.flag is properly set
+    const lastInstallFlag: RushLastInstallFlag = new RushLastInstallFlag(this._rushConfiguration);
+    if (!lastInstallFlag.isValid()) {
+      return Promise.reject(
+        new Error('Cannot perform "rush link" because the last install flag is missing.'
+          + ' Did you forget to run "rush install"?'));
```

Comment:
>Did you forget to run "rush install"? [](start = 14, length = 37)

I would phrase this as 'Did you run "rush install'?"

Sounds a little less accusatory.